cmd/xl-v1-multipart.go

```diff
@@ -200,12 +200,9 @@ func (xl xlObjects) newMultipartUpload(ctx context.Context, bucket string, objec
 
 	// If not set default to "application/octet-stream"
 	if meta["content-type"] == "" {
-		contentType := "application/octet-stream"
-		if objectExt := path.Ext(object); objectExt != "" {
-			content, ok := mimedb.DB[strings.ToLower(strings.TrimPrefix(objectExt, "."))]
-			if ok {
-				contentType = content.ContentType
-			}
+		contentType := mimedb.TypeByExtension(path.Ext(object))
```

Comment:
we should return to `application/octet-stream`

Reply:
Done